README.md

```diff
@@ -1,11 +1,37 @@
+
 # cachedcomplete
+
 Fast semi-static autocomplete for python
 
-## Normal Usage
-The normal usage is the same as in argcomplete, only changing the import from argcomplete to cachedcomplete
+## Installation
+
+Use the package manager pip to install cachedcomplete.
```

Comment:
Consider adding instructions for explicitly running "`setup.py`".
It's common practice with many packages